experimental/Schemes/src/WeilDivisor.jl

```diff
@@ -479,13 +538,53 @@ function _self_intersection(I::AbsIdealSheaf)
   return get_attribute(I, :_self_intersection)::Int
 end
 
+function _is_known_to_be_one(I::AbsIdealSheaf, U::AbsAffineScheme;
```

Comment:
May I suggest to name this function 
`has_is_one(I::AbsIdealSheaf, U::AbsAffineScheme)` 
and in its body to call
`has_is_one(I::Ideal)` ? 
We have the `has_is_...` syntax already in some other places. 
It means: has cached the value already ( or it is trivial to compute).

Reply:
I saw `has_is_one` being introduced. However, it is something different: Here we ask whether an ideal sheaf is known to be isomorphic to the unit ideal sheaf. `has_is_one` is asking whether it's easy to decide this question. The implementations are different and it would be more work to adapt this here. Since it's an internal function, I suggest to leave this for the moment.